src/PluginSuppression.php

```diff
@@ -491,4 +493,40 @@ private function is_callback_plugin_suppressed( $callback, $suppressed_plugins )
 			in_array( $source['name'], $suppressed_plugins, true )
 		);
 	}
+
+	/**
+	 * Add meta if plugin is suppressed in AMP page.
+	 *
+	 * @param array  $plugin_meta An array of the plugin's metadata.
+	 * @param string $plugin_file Path to the plugin file relative to the plugins directory.
+	 * @param array  $plugin_data An array of plugin data.
+	 *
+	 * @return array An array of the plugin's metadata
+	 */
+	public function filter_plugin_row_meta( $plugin_meta, $plugin_file, $plugin_data ) {
+
+		$screen = get_current_screen();
+
+		if ( ! ( $screen instanceof WP_Screen && 'plugins' === $screen->id ) ) {
```

Comment:
What was this needed for? Wouldn't the `plugin_row_meta` filter only be shown on the plugins screen anyway?

Reply:
I guess for the @delawski's comment in https://github.com/ampproject/amp-wp/pull/6959#pullrequestreview-903236852?